src/adv.py

```diff
@@ -37,15 +33,28 @@
 # Main
 #
 
-# Make a new player object that is currently in the 'outside' room.
-p = player("tom", "outside")
+
+def p_move(p, direction):
+    direction = direction + "_to"
+    p.current_room = getattr(p.current_room, direction)
```

Comment:
The fact that you use getattr() tells me you have a really good understanding of what is going on.  Good job!

Reply:
I was lucky finding it by googling ^^